### PR TITLE
Add extension TemplateDataRepresentable for URL

### PR DESCRIPTION
**Pull request by alirp88**
Adds support for serializing `URL`s in templates (#38).

### CODE REVIEW

File: Sources/TemplateKit/Data/TemplateDataRepresentable.swift [hunk 1]
```diff
@@ -89,3 +89,9 @@ extension Date: TemplateDataRepresentable {
         return .double(timeIntervalSince1970)
     }
 }
+
+extension URL: TemplateDataRepresentable {
+    public func convertToTemplateData() throws -> TemplateData {
```
Comment (tanner0101) [inline Sources/TemplateKit/Data/TemplateDataRepresentable.swift:95]: All public methods should have doc blocks. For methods that simply conform to a protocol, we usually add something like:
```swift
/// See `TemplateDataRepresentable`.
public func convertToTemplateData() throws -> TemplateData { ... }
```

File: Tests/TemplateKitTests/TemplateDataEncoderTests.swift [hunk 2]
```diff
@@ -59,6 +59,14 @@ class TemplateDataEncoderTests: XCTestCase {
             "hello": .string("hello"),
         ]))
     }
+    
+    func testEncodableWithURL() {
```
Comment (tanner0101) [inline Tests/TemplateKitTests/TemplateDataEncoderTests.swift:63]: This test needs to be added to the `allTests` variable (bottom or top of the file) to be run on Linux.

**Review by tanner0101 — COMMENTED** [review]
Change looks good :) Just some small things that need fixed. Thanks!

File: Tests/TemplateKitTests/TemplateDataEncoderTests.swift (hunk 2 above)
Comment (alirp88) [reply Tests/TemplateKitTests/TemplateDataEncoderTests.swift:63]: Yes, I'm new to this so I always forget it

Added

File: Sources/TemplateKit/Data/TemplateDataRepresentable.swift (hunk 1 above)
Comment (alirp88) [reply Sources/TemplateKit/Data/TemplateDataRepresentable.swift:95]: Done

**Pull request closed without merging**